filters/include/pcl/filters/convolution.h

```diff
@@ -72,7 +72,7 @@ namespace pcl
       */
 
     template <typename PointIn, typename PointOut>
-    class Convolution
+    class PCL_EXPORTS Convolution
```

Comment:
Can we use this instead on the functions directly? (Like what you did for `BSplineElements`)

Reply:
I wanted to do all the functions at once. That had the advantage that if you want to move more functions you don't have to think about it anymore. Now I have changed that. When moving functions you have to pay attention to PCL_EXPORTS yourself.